Prog/vacant-lots-fairfax.Rmd

```diff
@@ -362,65 +330,92 @@ zon <- st_transform(zon, crs = 4326)
 
 ```
 
+
+
+
+  
 Convert vacant lots to SF, set CRS. To convert to SF, we need to use only the observations with non-missing longitude and latitude. We separate these off and recombine after the spatial join.
 
+
+
 ```{r convert-sf}
 
-vacantsf <- vacant1 %>% 
-  filter(missing_coord == 0) %>% 
+jursf <- jur_3 %>% 
+  filter(missing_coord==0) %>% 
   st_as_sf(coords = c("long", "lat")) %>% 
   st_set_crs(st_crs(zon))
 ```
 
-Plot to make sure they are overlapping
-```{r make-map, echo=FALSE}
-# ggplot() +
-#   geom_sf(data = zon, mapping = aes()) +
-#   #scale_fill_manual(values = c("#1696d2", "#fdbf11", 
-#                                #"#d2d2d2", "#ec008b", 
-#                                #"#55b748")) +
-#   geom_sf(data = vacantsf, mapping = aes(),
-#           color = "black")
+
+
+Filter out Herndon and Vienna as these jurisdictions do not fall under Fairfax County zoning.
```

Comment:
do we need to get rid of Clifton too here? I thought they were missing in the zoning layer but in jursf_temp1 there are Clifton addresses (which admittedly could be not in "clifton" formally) that have zoning filled in.

Reply:
I'll remove Clifton too.